### DISCUSSION TITLE
Mi práctica JS

### DISCUSSION THREAD

**Discussion by Drako9159** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#443] [Ideas]
Ya con algo de responsive  https://drako9159.github.io/JS/dex.html

**Comment by Ingrzs** [comment]
Te quedo excelente, y como entre desde el celular de quedo muy ala izquierda,no pude escribir en el input bien 😅

**Reply by Drako9159** [reply]
Oh, jaja, me falta el responsive que haré más tarde
